src/fdb5/toc/TocPurgeVisitor.cc

```diff
@@ -112,8 +137,27 @@ void TocPurgeVisitor::report(std::ostream& out) const {
     if (!cnt2) {
         out << "    - NONE -" << std::endl;
     }
+    out << std::endl;
 
+    // Auxiliary files
+    out << "Auxiliary files to be deleted:" << std::endl;
+    for (const auto& it : deleteAuxFiles_) {
+        out << "    " << it << std::endl;
+    }
+    if (deleteAuxFiles_.empty()) {
+        out << "    - NONE -" << std::endl;
+    }
     out << std::endl;
+
+    out << "Auxiliary files to be kept:" << std::endl;
```

Comment:
No, we don't just list what we are going to remove: all of the .data files in dataUsage_ are listed, alongside their usage (0 if to be deleted, N otherwise).  As such, I list all of the aux files related to these files.